codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocolTestGenerator.kt

```diff
@@ -511,16 +534,11 @@ class ServerProtocolTestGenerator(
             FailingTest(RestJson, "RestJsonNoInputAndNoOutput", Action.Response),
             FailingTest(RestJson, "RestJsonNoInputAndOutputWithJson", Action.Response),
             FailingTest(RestJson, "RestJsonSupportsNaNFloatInputs", Action.Request),
-            FailingTest(RestJson, "RestJsonStreamingTraitsWithBlob", Action.Request),
-            FailingTest(RestJson, "RestJsonStreamingTraitsWithNoBlobBody", Action.Request),
-            FailingTest(RestJson, "RestJsonStreamingTraitsWithBlob", Action.Response),
-            FailingTest(RestJson, "RestJsonStreamingTraitsWithNoBlobBody", Action.Response),
-            FailingTest(RestJson, "RestJsonStreamingTraitsRequireLengthWithBlob", Action.Request),
-            FailingTest(RestJson, "RestJsonStreamingTraitsRequireLengthWithNoBlobBody", Action.Request),
+            FailingTest(RestJson, "RestJsonSupportsNaNFloatInputs", Action.Response),
+            FailingTest(RestJson, "RestJsonSimpleScalarProperties", Action.Response),
+            FailingTest(RestJson, "RestJsonSupportsInfinityFloatInputs", Action.Response),
+            FailingTest(RestJson, "RestJsonSupportsNegativeInfinityFloatInputs", Action.Response),
```

Comment:
Yes, earlier today I had pushed to this branch a commit merging from `main` https://github.com/awslabs/smithy-rs/commit/907c0f3ff9c10c44d946d25a5af5069beab45e43 (which removes these now-passing tests), so that you wouldn't have to deal with conflicts.

But you didn't fetch it (?), and you force-pushed the latest commit.

In general once a PR is open I _think_ it's best to _not_ rebase and force-push, only merge from main. Because AFAIK GitHub's UI won't show the previous history, only the rewrriten one, and that might confuse other people looking at the PR. The downside is that merging makes the history non-linear, but we always squash before merging, so I think it's a small price to pay for having an entire's PR history.

---

Anyway, these tests can be removed from the failing list.

Reply:
got it. thanks. pushed a commit to remove those.